http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonBPayloadIT.java

```diff
@@ -6,7 +6,7 @@
 public class JsonBPayloadIT extends JsonPayloadIT {
```

Comment:
I'm thinking but should this class deleted as well and `@QuarkusApplication` moved to `JsonPayloadIT` as it just extend one class. At least I would prefere to have single class for testing if the test are not reused in multiple test classes.